pkg/install/genevalogging.go

```diff
@@ -7,9 +7,11 @@ import (
 	"context"
 
 	"github.com/Azure/ARO-RP/pkg/genevalogging"
+	"github.com/Azure/ARO-RP/pkg/util/ensure"
 )
 
 func (i *Installer) ensureGenevaLogging(ctx context.Context) error {
-	gl := genevalogging.New(i.log, i.env, i.doc.OpenShiftCluster, i.kubernetescli, i.securitycli)
+	ensure := ensure.New(i.kubernetescli, i.securitycli)
```

Comment:
this is an anti-pattern because it prevents unit testing using mocks.  Create a field i.ensurer and instantiate it in NewInstaller.